test/sample_program/test_sample_program.py

```diff
@@ -0,0 +1,49 @@
+# This code is part of Qiskit.
```

Comment:
copyright should be `qiskit-runtime`. and I think this file can use some formatting